### DISCUSSION TITLE
Not able to use all the cores in CPU for web service

### DISCUSSION THREAD

**Discussion by mpraga95** [frePPLe/frepple#561] [Q&A]
Hi, I am unable to use all the cores in my system for frepple web service. i.e if I run the web service only one core is running at 100% remaining core usages are at minimum. For example I have a 16 core sytem how to make frepple web service use all the cores? can you help me with this?

**Comment by jdetaeye** [comment]
Answered already in https://github.com/frePPLe/frepple/discussions/466

It depends on the structure of the planning problem whether or not it can be run in threads.  The algorithm automatically detects subproblems that can be solved 100% independent of each other.
